_projects/ccc.md

```diff
@@ -4,8 +4,17 @@ collection: projects
 permalink: /projects/CCC
 abstract: "High-risk, novel, distributed computing exemplar project to demonstrate and investigate new ways of federating campus computing capabilities." 
 status: active
+logo: ccc_logo.jpg
 ---
 
-This project elaborates a plan to construct the Campus Compute Cooperative (CCC), a shared cyberinfrastructure as well as a social fabric. CCC will: (i) connect and federate campus physical infrastructures; (ii) use market-based mechanisms for resource allocation and quality of service to avoid the "tragedy of the commons"; (iii) leverage [InCommon](https://www.incommon.org/) and local identity management systems to provide federated authentication and authorization to resources; (iv) facilitate secure data and storage sharing between institutions and research labs; (v) offer "cloud bursting" to member institutions; and, (vi) provide paid-for, differentiated quality of service. In this way, CCC will both accelerate discovery and provide a research cyberinfrastructure with better cost/benefit by rationally and securely sharing the limited computational and human resources available for research enterprise.
-
-<a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1747813"><i class="fa fa-nsf"></i>NSF Award</a>
+<div class="project-description">
+	<figure>
+		<img src="../images/projects/ccc.jpg" width="300">
```

Comment:
Here the user will not need to add the exact name. The page can pick it up from the header
```suggestion
		<img src="{{site.baseurl}}/images/projects/{{page.logo}}" width="250">
```